assets/elm/Resolver/View.elm

```diff
@@ -1,24 +1,24 @@
 module Resolver.View exposing (view)
 
 import Html exposing (..)
-import Html.Attributes exposing (style, alt, href)
+import Html.Attributes exposing (alt, href, style)
 import Html.Events exposing (onClick)
 import Maybe exposing (withDefault)
-import Terms.Models exposing (Terms)
-import Target.Models exposing (DataSource)
+import Resolver.Helper as RH
+import Resolver.Messages exposing (Msg(..))
```

Comment:
possible to omit all of the changes to this file? rebasing against my resolver work would be much easier if that was the case.